### ISSUE TITLE
ERROR: The process "nnnn" not found.

### ISSUE THREAD

**Issue by arvy-p** [aws-azure-login/aws-azure-login#48]
I followed all the steps to install aws-azure-login. I then configured it (as shown below):
![image](https://user-images.githubusercontent.com/37854197/46057377-ba532100-c198-11e8-9dfa-5e91204050f1.png)

But when I attempt to login, I get an error and its not successfully logging in. Screenshots below:
![image](https://user-images.githubusercontent.com/37854197/46057532-5da43600-c199-11e8-8e55-7c3f4fd82264.png)

Can you please find the underlying issue ?

**Comment by mtrcn** [comment]
I have the same problem, does anyone know the solution?

**Comment by topgenorth** [comment]
Running into the same issue as well.  This is on Windows 10, running Powershell.  I've already run *aws-azure-login --configure*

**Comment by gitsno** [comment]
I get the error too, but my credentials are correctly saved. arvy-p I think your issue is that you have the wrong value for the URI - it should be a URI, like https://signin.aws.amazon.com/saml but you have something that looks like another ID (e4f4a434...)

**Comment by RipFence** [comment]
+1 on this issue running on Windows 10

**Comment by jeroen-nijssen** [comment]
+1 on this issue running on Windows 10

**Comment by sbuckpesch** [comment]
+1 on this issue running on Windows 10

**Comment by Wraiith32** [comment]
My whole team gets this error as well.  It appears safe to ignore because everything works but it is just annoying.  I've tried to look through the code and cannot find anything that might generate this.  Could be a dependency library or something causing it.  Not sure.

**Comment by justnealpatel** [comment]
@arvy-p were you able to verify what your issue was? I'm running into the same issue and my azure app uri is also a long id similar to the one you have.

**Comment by kennethaasan** [comment]
Is this still a problem with the latest version? I'm on Mac/Linux myself, so I would happily accept a PR that fixes this for Windows users.

**Comment by RipFence** [comment]
Yes, still a problem. The two things I’ve found that fix the problem temporarily are either deleting the credentials file and calling AWS-azure-login using its full path (no, I don’t have multiple copies installed).

**Comment by GavinEke** [comment]
I also still get this issue.

**Comment by kennethaasan** [comment]
Windows issue. Can you try to pull master and run it with debug? I'm on Mac and Linux so I'm not able to help :/

**Comment by GavinEke** [comment]
```
...
  aws-azure-login Request: https://signin.aws.amazon.com/saml +539ms
  aws-azure-login Received SAML response, browser closed +375ms
  aws-azure-login Finished state: username input +2ms
  aws-azure-login Found SAML response {lots of text} +4ms
ERROR: The process "14220" not found.
  aws-azure-login Converting assertion from base64 to ASCII +259ms
  aws-azure-login Converted {lots of text} +2ms
...
  aws-azure-login Asking user to choose role +7ms
? Role: arn:aws:iam::{hidden}
? Username: {my email}
? Session Duration Hours (up to 12): (8)   aws-azure-login Focusing on username input +3s
```

Not sure if this helps but it appears to happen after the SAML response and before the converting assertion.

**Comment by GavinEke** [comment]
So after spending the day troubleshooting.

The following line causes ERROR: The process "nnnn" not found. - https://github.com/sportradar/aws-azure-login/blob/master/lib/login.js#L530

I guess the bigger issue for myself and other Windows users are it just stops and breaks after prompting for the role.

```
  aws-azure-login Asking user to choose role +4ms
? Role: arn:aws:iam::{removed}
? Username: {my email}
? Session Duration Hours (up to 12): (8)   aws-azure-login Focusing on username input +2s
```

While interestingly if is use `--mode gui` it works and writes the ini file. Not sure if this is anything to do with the username prompt and the fact we use Seamless SSO so the username is not required.

```
  aws-azure-login Asking user to choose role +3ms
? Role: arn:aws:iam::{removed}
? Session Duration Hours (up to 12): 8
Assuming role arn:aws:iam::513217904333:role/TEST_ADMIN
  aws-azure-login Loading 'credentials' file at 'C:\Users\username\.aws\credentials' +12s
  aws-azure-login Parsing data +2ms
  aws-azure-login Setting credentials for profile 'default' +1ms
  aws-azure-login Stringifying credentials INI data +0ms
  aws-azure-login Creating AWS config directory 'C:\Users\username\.aws' if not exists. +1ms
  aws-azure-login Writing 'credentials' INI to file 'C:\Users\username\.aws\credentials' +1ms
```

**Comment by GavinEke** [comment]
So after spending the day troubleshooting.

The following line causes ERROR: The process "nnnn" not found. - https://github.com/sportradar/aws-azure-login/blob/master/lib/login.js#L530

I guess the bigger issue for myself and other Windows users are it just stops and breaks after prompting for the role.

```
  aws-azure-login Asking user to choose role +4ms
? Role: arn:aws:iam::{removed}
? Username: {my email}
? Session Duration Hours (up to 12): (8)   aws-azure-login Focusing on username input +2s
```

While interestingly if is use `--mode gui` it works and writes the ini file. Not sure if this is anything to do with the username prompt and the fact we use Seamless SSO so the username is not required.

```
  aws-azure-login Asking user to choose role +3ms
? Role: arn:aws:iam::{removed}
? Session Duration Hours (up to 12): 8
Assuming role arn:aws:iam::{removed}
  aws-azure-login Loading 'credentials' file at 'C:\Users\username\.aws\credentials' +12s
  aws-azure-login Parsing data +2ms
  aws-azure-login Setting credentials for profile 'default' +1ms
  aws-azure-login Stringifying credentials INI data +0ms
  aws-azure-login Creating AWS config directory 'C:\Users\username\.aws' if not exists. +1ms
  aws-azure-login Writing 'credentials' INI to file 'C:\Users\username\.aws\credentials' +1ms
```
